content/kb/troubleshooting-unable-to-mount-an-nfs-backup-target.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Troubleshooting: Unable to access an NFS backup target"
+author: James Lu
+draft: false
+date: 2022-05-27
+categories:
+  - "backup"
+---
+
+## Applicable versions
+
+All Longhorn versions.
+
+## Prerequisite
+
+Longhorn only supports NFS v4.0, v4.1 and v4.2.
+
+### How to determine which NFS version is used
+
+* Server side:
+
+The NFS server can support multiple versions by the configuration, so using which version for each connection will actually depend on how the NFS share is mounted by the client.
+
+What versions the server is configured to support:
+
+```bash
+  # cat /proc/fs/nfsd/versions
+```
+
```

Comment:
Simply mentioning the purpose of the command is enough.

```suggestion
Check which versions of NFS are currently enabled.

```bash
  # /proc/fs/nfsd/versions
```

```

Reply:
@derekbit could you take a look again please?